integration-tests/src/test/java/oracle/weblogic/kubernetes/ItKubernetesEvents.java

```diff
@@ -374,11 +374,44 @@ public void testDomainK8sEventsScalePastMax() {
     }
   }
 
+  /**
+   * Scale down and scale up the domain and verify that
+   * DomainProcessingCompleted normal event is generated.
+   */
+  @Order(7)
+  @Test
+  @DisplayName("Test domain completed event when domain is scaled.")
+  public void testScaleDomainAndVerifyCompletedEvent() {
+    try {
+      scaleDomainAndVerifyCompletedEvent(1, "scale down", true);
+      scaleDomainAndVerifyCompletedEvent(2, "scale up", true);
+    } finally {
+      scaleDomain(2);
+    }
+  }
+
+  private void scaleDomainAndVerifyCompletedEvent(int replicaCount, String testType, boolean verify) {
+    DateTime timestamp = new DateTime(Instant.now().getEpochSecond() * 1000L);
```

Comment:
I am not sure if this timestamp is too coarse-grained to make the verification of the event's timestamp after the two scale operations reliable. We could also get the event's count before and after each scale operation, and make sure that the count is increased by 1 each time.

Reply:
We get the current timestamp after each scaling operation. The test logic is as follows -
1. Get the current `timestamp`.
2. Scale down the domain 
3. Verify that `DomainProcessingComplete` event is generated after `timestamp` from step 1.
4. Get the current `timestamp` again.
5. Scale up the domain
6. Verify that `DomainProcessingComplete` event is generated after `timestamp` from step 4.

I thought we follow a similar validation approach in other test methods and am not sure if getting the event count is necessary.

Reply:
My concern is that the timestamp at step #1 and step #4 could be the same since the timestamp only gets the seconds. Either way, adding count in the verification is a good idea. I don't see it is tested anywhere.

Reply:
I have adding count verification in b0079f4